### ISSUE TITLE
Multiple fields selected

### ISSUE THREAD

**Issue by utiq** [godbasin/angular-select2#10]
Awesome, it works fine, but it doesn't work when I need to select multiple items by default. ngModel only allows an id.

For example:

```
<select2 [options]="exampleData" [(ngModel)]="multipleItems" [settings]="{ multiple: true }"></select2>
```

Where multipleItems is an array of Ids, for example:
```
constructor() {
 const multipleItems = ['59cfbc7ede2b2ccad9000000', '59d29f61de2b2cf988000000']
}
```

**Comment by utiq** [comment]
And the same happens in the onSelect event, it only gets the selected item, not the array of selected items.

**Comment by godbasin** [comment]
fixed by version 1.1.0
